### ISSUE TITLE
Test set effect

### ISSUE THREAD

**Issue by Wkhj11** [useracc687/saw-monodetr#1]
Thank you for your contribution. How can we improve the effectiveness of the Kitti test set in official evaluations? The evaluation results of my test set are significantly different from those of the local validation set

**Comment by tranquan687** [comment]
> Thank you for your contribution. How can we improve the effectiveness of the Kitti test set in official evaluations? The evaluation results of my test set are significantly different from those of the local validation set

Sorry for the late reply, you must train the model with the tranval split to evaluate on the test set.

**Comment by Wkhj11** [comment]
> > 感谢您的贡献。如何提高 Kitti 测试集在官方评估中的有效性？我的测试集评估结果与本地验证集的评估结果存在显著差异。
> 
> 抱歉回复晚了，您必须使用 tranval split 来训练模型，以便在测试集上进行评估。

Thank you very much for your reply. I will try again to divide the files according to your suggestions
